src/err/mod.rs

```diff
@@ -868,7 +901,7 @@ impl std::fmt::Debug for PyErr {
         Python::with_gil(|py| {
             f.debug_struct("PyErr")
                 .field("type", &self.get_type_bound(py))
-                .field("value", self.value(py))
+                .field("value", &self.value_bound(py))
```

Comment:
Hangover from before we changed `value_bound` to return a reference, I think:

```suggestion
                .field("value", self.value_bound(py))
```